app/models/color_scheme.rb

```diff
@@ -153,4 +153,6 @@ def dump_hex_cache
 #  version      :integer          default(1), not null
 #  created_at   :datetime         not null
 #  updated_at   :datetime         not null
+#  via_wizard   :boolean          default(FALSE), not null
```

Comment:
This doesn't look like it's related to that commit. Can you add them in another commit instead?